lib/bindings/bindings.cc

```diff
@@ -192,12 +192,10 @@ BindingInstance::bind_value(const char *name, int value)
     lua_pop(this->lua, 1);
     Debug("lua", "isnil? %s", isnil ? "yes" : "no");
 
-    if (isnil) {
-      lua_pushstring(this->lua, start);
-      lua_pushvalue(this->lua, value);
-      lua_settable(this->lua, -3);
-      bound = true;
-    }
+    lua_pushstring(this->lua, start);
```

Comment:
The comment above ("Only push the value if it is currently nil.") is no longer true after this change.  Please update with a new comment that describes that we always push the value so that it can be updated.